Sources/StreamVideoSwiftUI/CallView/LayoutComponents/HorizontalParticipantsBarView.swift

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import SwiftUI
+
+/// `BottomParticipantsBarLayoutComponent` represents a horizontally scrollable view of participant thumbnails.
+/// This component lays out participant thumbnails in a bar at the bottom of the associated view.
+public struct HorizontalParticipantsBarView<Factory: ViewFactory>: View {
```

Comment:
The "Bar" part still bugs me 😅 Usually we call bars other UI elements, maybe "list" instead?